src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(obj, value) {
+      obj.result += value;
+    },
+
+    subtract(obj, value) {
+      obj.result -= value;
+    },
+
+    multiply(obj, value) {
+      obj.result *= value;
+    },
+
+    divide(obj, value) {
+      obj.result /= value;
+    },
```

Comment:
let's use `this` instead of the `obj`, it must improve the reusability of the function

Reply:
![Uploading image.png…]()

Reply:
![image](https://github.com/mate-academy/js_advanced_calculator/assets/103724609/2e65b0b6-3db2-449e-b500-3eacc549e157)

Reply:
oh, sorry, you sent the obj as the argument